CHANGELOG/CHANGELOG-0.6.md

```diff
@@ -1,3 +1,25 @@
+## v0.6.1
+
+Changes Since `v0.6.0`:
+
+### Feature
+
+- Added MultiKueue  worker connection monitoring and reconnect. (#1809, @trasc)
```

Comment:
```suggestion
- Added MultiKueue worker connection monitoring and reconnect. (#1809, @trasc)
```

nit. 
I have already applied this improvement to each PR.

Reply:
Done